src/components/Banner/VaccinationsBanner.style.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components';
+import MuiButton from '@material-ui/core/Button';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+import { COLOR_MAP } from 'common/colors';
+
+export const Wrapper = styled.div`
+  background-color: ${COLOR_MAP.LIGHTGRAY_BG};
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem 1.5rem 1.75rem;
+  align-items: center;
+  justify-content: center;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    flex-direction: row;
+    padding: 2.5rem 1rem;
+  }
+`;
+
+export const Button = styled(MuiButton).attrs(props => ({
+  disableRipple: true,
+  disableFocusRipple: true,
+  variant: 'contained',
+}))`
+  font-size: 13px;
+
+  &:first-of-type {
+      background-color: ${COLOR_MAP.BLUE};
+      color: white;
+      box-shadow: box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.12);
+  }
+
+  &:last-of-type {
+    background-color: ${COLOR_MAP.LIGHTGRAY_BG};
+    color: ${COLOR_MAP.BLUE};
+    box-shadow: none;
+    margin-left: .5rem;
+    &:hover {
+        text-decoration: underline;
+    }
+}
+  `;
```

Comment:
It might be better to use 2 separate buttons and style them independently - the second button looks like it could use `variant="text"` https://material-ui.com/components/buttons/#text-buttons